_posts/2015-10-08-WhatAreIntervals.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "What Are Intervals?"
+author: "Rob Gillan"
+categories: terminology
+tags: [interval, intervals, learning, terminology, piano, guitar, stringed, instruments, instrument]
+img: piano-keys.png
+---
+At its most basic definition, an interval is simply the distance between two notes.
+
+In music, we name these distances because intervals mean a lot to us and names help give things context. Let's start with an example.
+
+The C-Major scale, which is one of the first things musicians are often taught, goes: `C D E F G A B C`. It's an easy beginner scale because it doesn't incorporate any sharps or flats and, on the piano, is played all on the white keys. Many people also learn as children of *solfege*: `Do Re Mi Fa So La Ti Do`.
+
+OK - I haven't actually talked about intervals yet. The most basic interval (and the interval that must exist in any equation) is the `root note`. The root note is the note that the scale or interval is based on -- in our above example, it's the first `C`. Technically speaking, the root note isn't an interval, but rather is the anchor for other intervals. Even more technically, the root note to the root note (in other words, the same note twice) is referred to as a `unison` interval.
+
+In the table below, I'll list each note in the C major scale and the named interval it represents in that scale.
+
+| Note #      | Note Name | Interval Name             |
```

Comment:
Yep. This works just fine in my test environment, which should emulate GFM almost perfectly (that's why I'm using `bundle exec` instead of plain ol' `jekyll serve`)